app.py

```diff
@@ -70,6 +71,44 @@
 
 app = Flask(__name__, static_url_path='')
 
+def into_number (string):
```

Comment:
You're looking for a hash function, and this not a great one because it will not distribute across the available key space very randomly.

Fortunately, Python has hash functions built in. You can get specific ones like md5 or sha256, but `hash()` provides a hashcode that Python itself uses for dictionaries:

```python
>>> hash('asdf')
6285382739108471212
```

The range is `-sys.maxint..sys.maxint`.

Note: this hash function will be different every time the Python program is restarted. That may or may not be what you want. Think about that, because your user may alternatingly be in or out of the treatment group every time the server is redeployed.

Use one of the hashes from `hashlib` or use a more common hash function if you need that.